src/containers/Asks/Asks.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { observer } from 'mobx-react'
```

Comment:
The observer it's not necessary can be delete because mobx is like redux for create stores and in this component not use store.

Reply:
done